app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def home
```

Comment:
we usually use a convention for naming controller actions that you can read more about here: http://docs.railsbridge.org/job-board/crud_and_resourceful_routing